Microsoft.Toolkit.Win32.UI.XamlHost/XamlApplication.Visibility.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What is this file for?

Reply:
To make XamlApplication public. Unfortunately this is a pattern that we follow for sharing code between the OS depot and toolkit depot.

---
In reply to: [260104926](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/85#discussion_r260104926) [](ancestors = 260104926)